pkgs/applications/editors/emacs/elisp-packages/manual-packages/ebuild-mode/default.nix

```diff
@@ -1,27 +1,36 @@
-{ lib, melpaBuild, fetchurl, writeText }:
+{
+  lib,
+  melpaBuild,
+  fetchgit,
+  writeText,
+}:
 
-melpaBuild rec {
-  pname = "ebuild-mode";
-  version = "1.67";
+let
+  argset = {
```

Comment:
This is just `rec` but harder to understand.

Reply:
I don't think this is harder than the conventional `rec`, since it is more explicit.
However, I will revert this.